tests/integration/regenerate.sh

```diff
@@ -114,7 +114,7 @@ run_test() {
     MAX_ITERATIONS=$MAX_ITERATIONS \
     DEFAULT_AGENT=$agent \
     TEST_RUNTIME="$TEST_RUNTIME" \
-    SANDBOX_CONTAINER_IMAGE="$SANDBOX_CONTAINER_IMAGE" \
+    SANDBOX_BASE_CONTAINER_IMAGE="$BASE_CONTAINER_IMAGE" \
```

Comment:
Should it not still be `"$SANDBOX_BASE_CONTAINER_IMAGE"` on the right (same for lines 188,215)?
Since it is potentially defaulted in line 61 above.